assets/xml/objects/object_horse_link_child.xml

```diff
@@ -9,27 +10,34 @@
         <Texture Name="object_horse_link_child_Tex_002168" OutName="tex_002168" Format="rgba16" Width="16" Height="16" Offset="0x2168" />
         <Texture Name="object_horse_link_child_Tex_002368" OutName="tex_002368" Format="rgba16" Width="16" Height="16" Offset="0x2368" />
         <Texture Name="object_horse_link_child_Tex_002568" OutName="tex_002568" Format="rgba16" Width="4" Height="4" Offset="0x2568" />
         <Texture Name="object_horse_link_child_Tex_002588" OutName="tex_002588" Format="rgba16" Width="8" Height="32" Offset="0x2588" />
         <Texture Name="object_horse_link_child_Tex_002788" OutName="tex_002788" Format="rgba16" Width="16" Height="16" Offset="0x2788" />
-        <Animation Name="object_horse_link_child_Anim_002F98" Offset="0x2F98" /> <!-- Original name is "hlc_anim_fastrun_30" -->
+
+        <Animation Name="gEponaGallopAnim" Offset="0x2F98" /> <!-- Original name is "hlc_anim_fastrun_30" -->
         <Animation Name="object_horse_link_child_Anim_0035B0" Offset="0x35B0" /> <!-- Original name is "hlc_anim_jump100" -->
         <Animation Name="object_horse_link_child_Anim_003D38" Offset="0x3D38" /> <!-- Original name is "hlc_anim_jump200" -->
-        <Animation Name="object_horse_link_child_Anim_0043AC" Offset="0x43AC" /> <!-- Original name is "hlc_anim_slowrun_30" -->
+        <Animation Name="gEponaTrotAnim" Offset="0x43AC" /> <!-- Original name is "hlc_anim_slowrun_30" -->
         <Animation Name="object_horse_link_child_Anim_004DE8" Offset="0x4DE8" /> <!-- Original name is "hlc_anim_stand" -->
         <Animation Name="object_horse_link_child_Anim_005F64" Offset="0x5F64" /> <!-- Original name is "hlc_anim_stop" -->
-        <Animation Name="object_horse_link_child_Anim_006D44" Offset="0x6D44" /> <!-- Original name is "hlc_anim_wait" -->
-        <Animation Name="object_horse_link_child_Anim_007468" Offset="0x7468" /> <!-- Original name is "hlc_anim_wait02" -->
-        <Animation Name="object_horse_link_child_Anim_007D50" Offset="0x7D50" /> <!-- Original name is "hlc_anim_walk_30" -->
+        <Animation Name="gEponaIdleAnim" Offset="0x6D44" /> <!-- Original name is "hlc_anim_wait" -->
+        <Animation Name="gEponaWhinnyAnim" Offset="0x7468" /> <!-- Original name is "hlc_anim_wait02" -->
+        <Animation Name="gEponaWalkAnim" Offset="0x7D50" /> <!-- Original name is "hlc_anim_walk_30" -->
+
         <!-- <Blob Name="object_horse_link_child_Blob_007D60" Size="0x2388" Offset="0x7D60" /> -->
+
+        <!-- z64Utils will not render because of skin type skeleton -->
         <Limb Name="object_horse_link_child_Skinlimb_00A0E8" Type="Skin" EnumName="OBJECT_HORSE_LINK_CHILD_LIMB_01" Offset="0xA0E8" />
         <Limb Name="object_horse_link_child_Skinlimb_00A0F8" Type="Skin" EnumName="OBJECT_HORSE_LINK_CHILD_LIMB_02" Offset="0xA0F8" />
         <Limb Name="object_horse_link_child_Skinlimb_00A108" Type="Skin" EnumName="OBJECT_HORSE_LINK_CHILD_LIMB_03" Offset="0xA108" />
```

Comment:
You could take a look at the dlists instead if you want, but it is fine if you don't want to

Reply:
sorry losing steam on this actor, gotta at least PR it before I forget about it like so many of my last documentation attempts.